module/zfs/multilist.c

```diff
@@ -104,7 +104,7 @@ void
 multilist_create(multilist_t *ml, size_t size, size_t offset,
     multilist_sublist_index_func_t *index_func)
 {
-	int num_sublists;
+	uint32_t num_sublists;
```

Comment:
Why 32 if the values around are either 64 or uint_t?

Reply:
It is an artifact of my treatment of `uint32_t` as a short hand for `unsigned int`. I will change it.